pkg/sfu/connectionquality/scorer.go

```diff
@@ -531,7 +536,11 @@ func scoreToConnectionQuality(score float64) livekit.ConnectionQuality {
 		return livekit.ConnectionQuality_GOOD
 	}
 
-	return livekit.ConnectionQuality_POOR
+	if score > 20.0 {
```

Comment:
nit: better to use const `cMinScore` here?

Reply:
Was just using numbers there. Will change it. Actually, can use 'cDisconnectedScore' which might be easier to read. Will make constants for others too in a separate PR.